rtengine/imageio.cc

```diff
@@ -83,6 +85,43 @@ Glib::ustring to_utf8 (const std::string& str)
     }
 }
 
+unsigned short addSat (unsigned short a, unsigned short b)
+{
+    unsigned short res = a + b;
+    res |= -(res < a);
+    return res;
+}
+
+void scanlineToPseudoGrey (const unsigned short *in16, unsigned char *out8, unsigned long width)
```

Comment:
Of course. But as I saw no `<cstdint>` include but the use of `unsigned short` I went for the latter.